tests/integration/modifiers/scroll-into-view-test.js

```diff
@@ -220,4 +220,92 @@ module('Integration | Modifier | scroll-into-view', function (hooks) {
       assert.ok(this.scrollToSpy.notCalled, 'scrollTo was not called');
     });
   });
+
+  module('with offsets and custom scroll container', function (offsetHooks) {
+    const sandbox = sinon.createSandbox();
```

Comment:
This isn't necessary. Sinon creates its own sandbox by default, which we clean up automatically here: https://github.com/elwayman02/ember-scroll-modifiers/blob/master/tests/test-helper.js#L13

Let's remove this `sandbox` variable and just call `Sinon.spy` directly. We won't need to call `restore()` either, as that's handled by the `setupSinon()` method linked above.

Reply:
I know there are other tests that explicitly create a sandbox, but those are also wrong. I just haven't had time to clean them up. 😓

Reply:
Good to know, removed the sandbox usages.